console/console-init/ui/src/modules/address/dialogs/CreateAddress/CreateAddress.tsx

```diff
@@ -139,6 +139,11 @@ export const CreateAddress: React.FunctionComponent = () => {
           if (deadletterAddress && deadletterAddress.trim() !== "") {
             variable.spec.deadletter = deadletterAddress.trim();
           }
+        }
+        if (
+          (addressType && addressType.trim().toLowerCase() === "queue") ||
```

Comment:
Please use enum type `AddressTypes` for values "queue", "topic" and "subscription"